### PR TITLE
Test app with two assigned USB devices

### PR DESCRIPTION
**Pull request by dautovri**
Simple test of 2 USB passthrough functionality after reboot of guest

Signed-off-by: Ruslan Dautov <dautov2@gmail.com>

### CODE REVIEW

**Comment by sadov** [comment]
@dautovri Was this test done for both KVM and XEN?

**Comment by dautovri** [comment]
@sadov, I developed a test to check this device on KVM first. Because we still have issues with Xen https://github.com/lf-edge/eden/issues/578. Xen versions will be fixed later.

**Comment by sadov** [comment]
OK. It seems to work on KVM for me.

File: tests/hardware_reboot/README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Test Description
 
 Key purpose is to verify that a reboot (sudo shutdown -r +1) from the guest doesn’t disrupt the set of assigned adapters.
+Tests working correctly only on KVM
```
Comment (mydatascience) [inline tests/hardware_reboot/README.md:4]: *work

**Pull request merged**